packages/office-ui-fabric-react/src/components/ComboBox/ComboBox.types.ts

```diff
@@ -1,311 +1,319 @@
-   * @default true
-   */
-  isButtonAriaHidden?: boolean;
-
-  /**
-   * Optional keytip for this combo box
-   */
-  keytipProps?: IKeytipProps;
-
-  /**
-   * Value to show in the input, does not have to map to a combobox option
-   * @deprecated Use `text` instead.
-   */
-  value?: string;
-}
-
-export interface IComboBoxStyles {
-  /**
-   * Style for the container which has the ComboBox and the label
-   */
-  container: IStyle;
-
-  /**
-   * Style for the label element of the ComboBox.
-   */
-  label: IStyle;
-
-  /**
-   * Style for the label element of the ComboBox in the disabled state.
-   */
-  labelDisabled: IStyle;
-
-  /**
-   * Base styles for the root element of all ComboBoxes.
-   */
-  root: IStyle;
-
-  /**
-   * Styles for the root element for variant of ComboBox with an errorMessage in the props.
-   */
-  rootError: IStyle;
-
-  /**
-   * Styles for variant of ComboBox where allowFreeForm is false in the props.
-   */
-  rootDisallowFreeForm: IStyle;
-
-  /**
-   * Styles for when the ComboBox is hovered. These styles are applied for all comboBoxes except when
-   * the comboBox is disabled.
-   */
-  rootHovered: IStyle;
-
-  /**
-   * Styles for when the ComboBox is active. These styles are applied for all comboBoxes except when
-   * the comboBox is disabled.
-   */
-  rootPressed: IStyle;
-
-  /**
-   * Styles for when the ComboBox is focused. These styles are applied for all comboBoxes except when
-   * the comboBox is disabled.
-   */
-  rootFocused: IStyle;
-
-  /**
-   * Styles for when the comboBox is disabled. These styles override all the other styles.
-   * NOTE : Hover (or) Focused (or) active styles are not applied for disabled comboBoxes.
-   */
-  rootDisabled: IStyle;
-
-  /**
-   * Base styles for the input element - which contains the currently selected
-   * option.
-   */
-  input: IStyle;
-
-  /**
-   * Style override for the input element when comboBox is disabled.
-   */
-  inputDisabled: IStyle;
-
-  /**
-   * Styles for the error Message text of the comboBox.
-   */
-  errorMessage: IStyle;
-
-  /**
-   * Styles for the callout.
-   */
-  callout: IStyle;
-
-  /**
-   * Styles for the optionsContainerWrapper.
-   */
-  optionsContainerWrapper: IStyle;
-
-  /**
-   * Styles for the container of all the Combobox options
-   * Includes the headers and dividers.
-   */
-  optionsContainer: IStyle;
-
-  /**
-   * Styles for a header in the options.
-   */
-  header: IStyle;
-
-  /**
-   * Styles for a divider in the options.
-   */
-  divider: IStyle;
-}
-
-export interface IComboBoxOptionStyles extends IButtonStyles {
-  /**
-   * Styles for the text inside the comboBox option.
-   * This should be used instead of the description
-   * inside IButtonStyles because we custom render the text
-   * in the comboBox options.
-   */
-  optionText: IStyle;
-
-  /**
-   * Styles for the comboBox option text's wrapper.
-   */
-  optionTextWrapper: IStyle;
-}
+import { IIconProps } from '../../Icon';
```

Comment:
Why is this whole file changing as part of this PR?

Reply:
Looks like I somehow added a trailing whitespace to all the lines. I'll fix this. Thanks.